ui/helpers/http.js

```diff
@@ -8,11 +8,13 @@ const generateHeaders = function generateHeaders() {
   });
 };
 
+// TODO: The credentials param can be removed in July once Firefox 62 ships and it is the default.
 export const create = function postJson(uri, data) {
   return fetch(uri, {
     method: 'POST',
     headers: generateHeaders(),
     body: JSON.stringify(data),
+    credentials: "same-origin",
```

Comment:
single quotes in JS (double in JSX attributes)

(the quotes rule is still turned off in the linter - I'll see about getting it enabled soon)